### PR TITLE
Rename Hello World r=robjperez

### PR DESCRIPTION
**Pull request by Lucashuang0802**

### CODE REVIEW

File: 1.Basic-Video-Chat/Basic-Video-Chat/ViewController.swift [hunk 1]
```diff
@@ -9,9 +9,6 @@
 import UIKit
 import OpenTok
 
-let kWidgetHeight = 240
-let kWidgetWidth = 320
```
Comment (Lucashuang0802) [inline 1.Basic-Video-Chat/Basic-Video-Chat/ViewController.swift]: I replace it just to try to make this area as clean as possible so users can focus on `Fill the following variables using your own Project info` block.

**Comment by Lucashuang0802** [comment]
This change will be applied to the Objective-C one once it's merged. 😃

**Comment by robjperez** [comment]
LGTM

**Comment by jeffswartz** [comment]
Please change the name from 1.Hello-World to 1.Basics.

**Pull request merged**